shared/html/devtools-panel.html

```diff
@@ -92,6 +118,17 @@
       overflow: hidden;
     }
 
+    /* Make it easier to view contents of currently-hovered cell. */
+    td:hover {
+      white-space: break-spaces;
+      background-color: beige;
+      word-wrap: break-word;
+    }
+
+    tr:hover {
+      border: 1px solid black;
```

Comment:
This makes each line jump by one px even if it's not expanding in size. I think we could instead just change the shade of the tr background on highlight.

The alternative I think requires using transparent borders but I played around and forgot how to make this work :')